### PR TITLE
Add plop generator for components

### PR DESCRIPTION
**Pull request by drewdrewthis**
https://www.npmjs.com/package/plop

### CODE REVIEW

File: plopfile.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            validate: function (value) {
+                if ((/.+/).test(value)) { return true; }
+                return 'name is required';
+            }
+        },
+        {
+            type: 'input',
+            name: 'type',
+            message: "Is this an atom, molecule, organism, or template?",
+            validate: function (value) {
```
Comment (cisacke) [inline plopfile.js]: maybe also test here that the response is one of the options?

File: plop_templates/jsx_template.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <div className={ classnames(className, styles['{{lowerCase name}}']) }>
+      { children }
+    </div>
+  );
+}
+
+
+{{properCase name}}.propTypes = {
+  /**
+   * This prop will be add a new className to any inherent classNames
```
Comment (cisacke) [inline plop_templates/jsx_template.txt]: fix up the copy here

File: package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "open": "open http://localhost:6060",
+    "c": "yarn run plop component",
```
Comment (cisacke) [inline package.json]: could change this to something more descriptive

File: src/templates/Navigator/Readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Navigator Example:
```
Comment (cisacke) [inline src/templates/Navigator/Readme.md]: actually looks like you're using it in your next PR so keep it in

File: package.json (hunk 3 above)
Comment (drewdrewthis) [inline package.json]: @cisacke The `yarn run c` is just a shorthand. The full command is the description.

Comment (cisacke) [inline package.json]: 👍

**Pull request merged**